the-content-guide/readme.md

```diff
@@ -1,20 +1,108 @@
 
-## User story:
+This guide will describe what ingredients work well together and how you can best present the meal. It will allow you to attract your target audience and keep them maximally engaged. The guide will also explain how you can test your business metrics and how to reach your audience. Most of all, this guide will tell you how to create content you can be proud of!
 
-To know what makes good content — As a communication professional — I can read a comprehensive guide about content creation.
+## Originality & authenticity
+
+Good content is original. [Content Farms](https://en.wikipedia.org/wiki/Content_farm) are no longer wanted in these times. Search engines, such as Google, are getting better and better at detecting these 'clickbait sites' and these sites are not received well. Original content is sure to be shared more often. Moreover, why would you say something that's not worth saying?
+
+> You share content because you want to achieve a certain goal. You want it to be read, and you want the reader to learn something from it.
+
+Apart from originality, high quality content also makes a certain promise about the credibility of the text. When sources have been used they should provide authority. Good sources are crucial and ensure that a text is taken seriously.
+
+Be a guardian of the originality and authenticity of your content piece. It's easier said than done while we all scrape information from different kinds of sources. When it's authority in people you interview, articles, books or your own judgement, it's all based on other work. The difference where you as an author come in, is how you handle these sources. Combine them to share new insights.
+
+Repetition is not always a bad thing, it can be a powerful tool, but know that it will only work when it's used for original, compelling observations. High quality content is creative and has the potential to excite audiences.
+
+## Structure & readability
+
+Readers are looking for answers and want to find them in the easiest, fastest way possible. A well structured text gives guidance and leads the reader towards its conclusion in a gentle, organic fashion. A clear text consists of:
+
+* A title
+* An intro
+* Headers
+* **Bold** and *italic*  
+* Short paragraphs
+* Clear quotes
+* A conclusion
+
+A strong and remarkable *title* is important to attract your reader. A good title should activate, interest and excite. Often enough with newspaper articles—also the digital ones—the title is the key component in attracting the reader.
+
+If you have lured your reader with a fantastic title, he/she often decides whether to keep on reading based on the first few sentences. This brings us to the intro of the text and the importance thereof.
+
+> If the title and image are the bait, then the intro is the line that has to be strong enough to reel the reader in. And we all can all imagine how much it would suck to fish with a line that breaks every time you catch a fish. 
```

Comment:
Sorry dude:

'If the title and image are the bait, then the intro is the line that reels the reader in. We can all imagine how much it would suck to fish with a line that breaks every time you catch a fish, so that line needs to be strong.'